.circleci/config.yml

```diff
@@ -30,7 +30,8 @@ jobs:
       - checkout
       - restore_cache:
           key: v1-npm-{{ checksum "package-lock.json" }}
-      - run: npm ci
+      # TEMP: remove --legacy-peer-deps once storybook 6.5 is released
+      - run: npm ci --legacy-peer-deps
```

Comment:
`--legacy-peer-deps` is needed until storybook 6.5 is released. All dependecies we pull rely on and work with storybook `6.x.` so they will work after the release